packages/workspace/src/browser/quick-open-workspace.ts

```diff
@@ -48,45 +42,34 @@ export class QuickOpenWorkspace {
     };
 
     async open(workspaces: string[]): Promise<void> {
-        this.items = [];
-        const [homeDirUri] = await Promise.all([
-            this.envServer.getHomeDirUri(),
-            this.workspaceService.getUntitledWorkspace()
-        ]);
-        const home = new URI(homeDirUri).path.toString();
-        await this.preferences.ready;
```

Comment:
Do you know why this was awaited and why it's no longer necessary?

Reply:
Back in https://github.com/eclipse-theia/theia/commit/543b119a32652f662968411370979ea002a1736a#diff-52cd48457988091d685e5eb18c42cab1159c2a17740bd035d5cc1dd9ccd07464, it seems like the preferences were actually used. However, the usage was removed in https://github.com/eclipse-theia/theia/commit/3ac912faf25c298c4af4578311562d23dde257c0#diff-52cd48457988091d685e5eb18c42cab1159c2a17740bd035d5cc1dd9ccd07464, without cleaning up call to `ready`.